### PR TITLE
No auth warning

### PR DESCRIPTION
**Pull request by efajardo**

### CODE REVIEW

**Comment by efajardo** [comment]
@ddavila0 thanks I applied your changes and also the change requeste in [PR-45](https://github.com/opensciencegrid/xrootd-lcmaps/pull/45) so maybe we can close that one.

File: src/XrdLcmapsConfig.cc [hunk 1]
```diff
@@ -173,6 +174,12 @@ int XrdSecgsiAuthzConfig(const char *cfg)
          return -1;
       }
    }
+   else {
+     PRINT(warn_pfx << "Setting LCMAPS config file" << cfg_file << " won't be used, no-authz option is set.");
+     PRINT(warn_pfx << "Setting LCMAPS policy name " << policy_name << " won't be used, no-authz option is set.");
+     PRINT(warn_pfx << "Setting LCMAPS log level " << log_level << " won't be used, no-authz option is set.");
```
Comment (brianhlin) [inline src/XrdLcmapsConfig.cc:180]:
```suggestion
     PRINT(warn_pfx << "LCMAPS config file " << cfg_file << " won't be used: no-authz option is set.");
     PRINT(warn_pfx << "LCMAPS policy name " << policy_name << " won't be used: no-authz option is set.");
     PRINT(warn_pfx << " LCMAPS log level " << log_level << " won't be used: no-authz option is set.");
```

**Comment by bbockelm** [comment]
@efajardo - here's my thinking:
1.  Only do warning when conflicting options are set: that is, if a LCMAPS config file is provided _and_ `no-authz` is set, then warn the user.  As the two config variables conflict, they likely don't mean to do this.
2.  When `no-authz` is set (and nothing conflicts), then we should have an `info`-level message saying something like "The no-authz option is set; LCMAPS will not be loaded or invoked."

**Comment by bbockelm** [comment]
It appears that indention is messed up.  Can you see if perhaps your editor is mixing tabs and spaces?

Is it possible to separate out the switch to the Xrootd base version to 4.12?

**Comment by efajardo** [comment]
@bbockelm and @brianhlin I fixed all the problems can you approve. I will create a separate PR for 4.12

**Pull request merged**